makefiles/default_modules.deps.mk

```
@@ -0,0 +1,17 @@
+# This files contains dependencies for default modules. They are parsed at the
+# end of the dependency loop. They MAY inlcude new modules, but this modules
+# MUST NOT have dependencies themselfs.
+
+ifneq (,$(filter ztimer,$(USEMODULE)))
+  ifneq (,$(filter auto_init_ztimer,$(USEMODULE)))
+    USEMODULE += ztimer_init
+  endif
+endif
```

Shouldn't this only be 

```suggestion
ifneq (,$(filter auto_init_ztimer,$(USEMODULE)))
  USEMODULE += ztimer_init
endif
```

since `ztimer` is not a default module?